lib/components/settings/pages/customization/customization.dart

```diff
@@ -49,23 +49,33 @@ class _SettingsPageCustomizationState extends State<SettingsPageCustomization>
               width: double.infinity,
               child: Padding(
                 padding: const EdgeInsets.all(8.0),
-                child: Row(
-                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-                  children: SettingsPresets.themeModePresets
-                      .map((e) => ThemeModeButton(model: e))
-                      .toList(),
+                child: Align(
+                  child: SingleChildScrollView(
+                    scrollDirection: Axis.horizontal,
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                      children: SettingsPresets.themeModePresets
+                          .map((e) => ThemeModeButton(model: e))
+                          .toList(),
+                    ),
+                  ),
                 ),
               ),
             ),
             SizedBox(
               width: double.infinity,
               child: Padding(
                 padding: const EdgeInsets.all(8.0),
-                child: Row(
-                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-                  children: BuiltinColor.values
-                      .map((e) => AccentColorButton(color: e))
-                      .toList(),
+                child: Align(
+                  child: SingleChildScrollView(
+                    scrollDirection: Axis.horizontal,
```

Comment:
Actually, ListView forces the scroll effect even if the list is small and fits in the screen (Over scroll effect will be there) but Align + SingleChildScrollView dont have it

Reply:
You can use a custom physics class to remove the effect iirc, or just use the bounce scroll physics

Reply:
yes, but alignment is also a problem, along with that we need to have a fixed height for the ListView when scrolling horizontally to avoid vertical viewport error, etc

Reply:
Still, the widgets you use in such a configuration make the final result hard to predict as it's not really clear what each widget actually contributes to in the layout. I'd still prefer if you used another approach (constraints using SizedBox and for alignment either ignore or completely use another approach).